src/review/index.tsx

```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components'
+
+import { space } from '_utils/branding'
+import Review from './Review'
+
+const StyledReview = styled(Review)`
+  & {
+    padding: ${space.m} 0;
+  }
+
+  &.kirk-is-review-response {
+    padding-left: ${space.xxl};
```

Comment:
Are you sure it's not `xl`? Looks big to me in the screenshot.

Reply:
🦅👁

Reply:
Fixed to xl